server/controllers/article.js

```diff
@@ -114,10 +133,7 @@ export const deleteArticle = async (req, res) => {
   const { id } = req.params;
 
   try {
-    const article = await Article.findOne({
-      where: { id, userId }
-    });
-
+    const article = await findArticle(req);
```

Comment:
I don't think there is a need to go to the database every time you want to calculate reading time, you can retrieve the title and body from the initial DB instance and calculate directly